packages/usa.florida/README.md

```diff
@@ -0,0 +1,86 @@
+# 🇺🇸 USA, Florida
+
+[![License: Unlicense](https://img.shields.io/badge/license-Unlicense-blue.svg)](http://unlicense.org/)
+
+Contains all the counties of the Florida state:
+* Alachua
+* Baker
+* Bay
+* Bradford
+* Brevard
+* Broward
+* Calhoun
+* Charlotte
+* Citrus
+* Clay
+* Collier
+* Columbia
+* DeSoto
+* Dixie
+* Duval
+* Escambia
+* Flagler
+* Franklin
+* Gadsden
+* Gilchrist
+* Glades
+* Gulf
+* Hamilton
+* Hardee
+* Hendry
+* Hernando
+* Highlands
+* Hillsborough
+* Holmes
+* Indian River
+* Jackson
+* Jefferson
+* Lafayette
+* Lake
+* Lee
+* Leon
+* Levy
+* Liberty
+* Madison
+* Manatee
+* Marion
+* Marti
+* Miami-Dad
+* Monroe"
+* Nassau
+* Okaloosa
+* Okeechobee
+* Orange
+* Osceola
+* Palm Beach
+* Pasco
+* Pinellas
+* Polk
+* Putnam
+* Saint Johns
+* Saint Lucie
+* Santa Rosa
+* Sarasota
+* Seminole
+* Sumter
+* Suwannee
+* Taylor
+* Union
+* Volusia
+* Wakulla
+* Walton
+* Washington
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/usa.florida`
+
+### yarn
+
+`yarn add @svg-maps/usa.florida`
+
+## Credits
+
+This map is based on the work of [Dbenbenn](https://commons.wikimedia.org/wiki/User:Dbenbenn).
```

Comment:
Could you add a link to the original map and indicate the modifications you've done please (e.g. [USA map](https://github.com/VictorCazanave/svg-maps/tree/master/packages/usa))?